contracts/staking/GoodFundManager.sol

```diff
@@ -126,7 +128,14 @@ contract GoodFundManager is DAOContract {
 	// addRights();
 	//   super.start();
 	// }
-
+	/**
+	 * @dev Set gas cost to mint GD rewards for keeper
```

Comment:
that is cast cost for the whole mint ubi+bridge ubi+mint rewards no? not just the rewards minting cost